components/proxy/src/main/java/com/hotels/styx/applications/yaml/YamlApplicationsProvider.java

```diff
@@ -32,14 +33,16 @@
 import static com.hotels.styx.api.io.ResourceFactory.newResource;
 import static com.hotels.styx.client.applications.BackendServices.newBackendServices;
 import static java.lang.String.format;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.StreamSupport.stream;
 
 /**
  * Provides applications by reading from a YAML file.
  *
  */
 public class YamlApplicationsProvider implements ApplicationsProvider {
```

Comment:
Apart from tests, is the origins starter app only place where this class is used from? 

As such, can you investigate if it can be removed?